src/storage/src/test_util/config_util.rs

```diff
@@ -51,13 +52,15 @@ pub async fn new_store_config(
         ..Default::default()
     };
     let log_store = Arc::new(RaftEngineLogStore::try_new(log_config).await.unwrap());
-
+    let compaction_scheduler = Arc::new(NoopCompactionScheduler::default());
     StoreConfig {
         log_store,
         sst_layer,
         manifest,
         memtable_builder: Arc::new(DefaultMemtableBuilder::default()),
         flush_scheduler,
         flush_strategy: Arc::new(SizeBasedStrategy::default()),
+        compaction_scheduler,
+        engine_config: Arc::new(Default::default()),
```

Comment:
[`Arc` derives `Default`](https://doc.rust-lang.org/std/sync/struct.Arc.html#impl-Default-for-Arc%3CT%3E), if you want to change